core/src/main/java/com/amplifyframework/analytics/AnalyticsCategoryBehavior.java

```diff
@@ -33,7 +34,7 @@
      * @param userId The unique identifier for the user
      * @param profile User specific data (e.g. plan, accountType, email, age, location, etc)
      */
-    void identifyUser(@NonNull String userId, @NonNull AnalyticsProfile profile);
+    void identifyUser(@NonNull String userId, @Nullable UserProfile profile);
```

Comment:
If you make this nullable, add a note to the javadoc about what happens when it is null / what the null means.